project-fob/Controllers/AttendeeController.cs

```diff
@@ -26,21 +26,43 @@ public ActionResult Index()
 
         public void Fob(string value)
         {
-            HttpContext.Session.TryGetValue("meetingid", out var meetingIdValue);
+            if (HaveCookieId())
+            {
+                string userId = GetCookieId();
+                HttpContext.Session.TryGetValue("meetingid", out var meetingIdValue);
```

Comment:
How come session state is still being used with the cookies? Am I misunderstanding what this PR was for? I thought it was to remove session state.